<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperatura:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcular">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperatura:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>119</height>
     </rect>
    </property>
    <property name="title">
     <string>Convertir a:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="centigrados_kelvin">
       <property name="text">
        <string>Centigrados a Kelvin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="kelvin_centigrados">
       <property name="text">
        <string>Kelvin a Centigrados</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="centigrados_fahrenheit">
       <property name="text">
        <string>Centigrados a Fahrenheit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="fahrenheit_centigrados">
       <property name="text">
        <string>Fahrenheit a Centigrados</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="input_temp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="conversion">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
